README.md

```diff
@@ -178,6 +163,29 @@ then allowing dynamic injection of `<script type="importmap-shim">` to immediate
 
 This follows the [dynamic import map specification approach outlined in import map extensions](https://github.com/guybedford/import-maps-extensions).
 
+### Skip Processing
+
+> Stability: Non-spec feature
+
+When loading modules that you know will only use baseline modules features, it is possible to set a rule to explicitly
+opt-out modules from rewriting. This improves performance because those modules then do not need to be processed or transformed at all, so that only local application code is handled and not library code.
+
+This can be configured by setting the `skip` URL regular expression on a `importShimInitialOptions` global before loading es-module-shims:
+
+```js
+self.importShimInitialOptions = {
```

Comment:
How about calling it `esmShimsInitOptions = {...}`?

Or maybe `esmsInitOptions` or `ESMShimsInitOptions`?